tests/php/SiteTreeContentReviewTest.php

```diff
@@ -372,14 +373,30 @@ public function testPermissionCheckByOnDataObject()
     {
         $reviewer = $this->objFromFixture(Member::class, 'editor');
 
-        // Mock Page class with canReviewContent method to return true on first call and false on second call
-        $mock = $this->getMockBuilder(Page::class)
-            ->setMethods(['canReviewContent', 'NextReviewDate', 'OwnerUsers'])
-            ->getMock();
-        $mock->expects($this->exactly(2))->method('canReviewContent')->willReturnOnConsecutiveCalls(false, true);
-        $mock->method('NextReviewDate')->willReturn('2020-02-20 12:00:00');
-        $mock->method('OwnerUsers')->willReturn(ArrayList::create([$reviewer]));
-        $mock->ContentReviewType = 'Custom';
+        // Mock Page class with canReviewContent method to return false on first call and true on second call
+        $mock = new class extends Page {
```

Comment:
Move this class into its own file, it's bigger than a couple of lines

Reply:
Done